modules/installation-about-restricted-network.adoc

```diff
@@ -44,15 +47,15 @@ endif::[]
 In {product-title} {product-version}, you can perform an installation that does not
 require an active connection to the Internet to obtain software components. Restricted network installations can be completed using installer-provisioned infrastructure or user-provisioned infrastructure, depending on the cloud platform to which you are installing the cluster.
 
-ifndef::ibm-power[]
+ifndef::ibm-power,upi[]
```

Comment:
Was removing this snippet for all UPI assemblies intended? Last I heard, this was still a relevant note for UPI restricted networks.

Reply:
@codyhoag - This snippet is about adding (not removing) condition for upi assemblies. Earlier this content was visible only for ibm-power. I added UPI too for the content visibility condition. Sorry if I understood your comment incorrectly.

Reply:
@snarayan-redhat since this is an `ifndef`, it is hiding the content. I think you're thinking of an `ifdef`. You can see that it is currently showing (e.g., [here](https://docs.openshift.com/container-platform/4.8/installing/installing_aws/installing-restricted-networks-aws.html#installation-about-restricted-networks_installing-restricted-networks-aws)). So UPI will need to be removed here for it to display.

Reply:
@codyhoag - Thanks for pointing that out. I have made the changes. I can see that the section is getting rendered now.